packages/core/src/tw/view/pointer-observer.ts

```diff
@@ -66,6 +66,10 @@ export class PointerObserver implements IPointerObserver {
     }
 
     protected handleMouseDown = (event: MouseEvent) => {
+        const page = this.resolvePage(event);
+        if (page === null) {
```

Comment:
Hmm, I don't understand why this check is needed. `resolveCoordinates` should be expected to return an offset if one can be resolved, or return null otherwise.

What is the error that you see?

Reply:
Here is the issue: 
1. scroll down until toolbar fixes on top
2. select multiple lines
3. click on the toolbar

your selection gone